src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FileUploadsValidatorServiceTests.cs

```diff
@@ -255,22 +259,23 @@ await using (var context = InMemoryApplicationDbContext(contextId))
         [Fact]
         public async Task ValidateDataFilesForUpload_ReplacingDataFileWithFileOfSameName()
         {
-            var releaseId = Guid.NewGuid();
+            var release = _fixture.DefaultRelease()
+                .Generate();
+
+            // The file being replaced here has the same name as the one being uploaded, but that's ok.
+            var fileBeingReplaced = _fixture.DefaultFile()
```

Comment:
I don't think I can because one is a `File` type and one is a `ReleaseFile`.

The reason for declaring `fileBeingReplaced` separately rather than inlining it into the declaration of `releaseFile` is so that it can be commented on with the code comment, and so that it can be named as `fileBeingReplaced`.

It's used as the `replacingFile` argument later on in the `ValidateDataFilesForUpload` call.

Reply:
Oh so it is!  Thanks.